packs/kingmaker-bestiary/nugrah.json

```
@@ -1404,7 +1404,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>Prerequisite Nugrah has a summon animal spell prepared and available.</p>\n<hr />\n<p><strong>Effect</strong> Nugrah casts summon animal but does not need to use a material component.</p>"
+                    "value": "<p><strong>Prerequisite</strong> Nugrah has a @UUID[Compendium.pf2e.spells-srd.Item.Summon Animal] spell prepared and available.</p><hr /><p><strong>Effect</strong> Nugrah casts <em>summon animal</em> but does not need to use a material component.</p>"
```

```suggestion
                    "value": "<p><strong>Requirements</strong> Nugrah has a @UUID[Compendium.pf2e.spells-srd.Item.Summon Animal] spell prepared and available.</p><hr /><p><strong>Effect</strong> Nugrah casts <em>summon animal</em> but does not need to use a material component.</p>"
```
We can at least clean up Paizo's typo